### DISCUSSION TITLE
kitty got stuck with two lines at the top in the terminal

### DISCUSSION THREAD

**Discussion by laoshaw** [kovidgoyal/kitty#4335] [General]
On ubuntu 20.04 with `apt install kitty` , when I launch kitty, it sometimes got stuck with only two lines shown up at the top of terminal, e.g. if I do a `ls -l` with 100 files listed, it will scroll within the top two lines and the cursor did not proceed to the bottom of kitty terminal. It does not occur all the time, but quite often.

**Comment by laoshaw** [comment]
The way I got out of it is I 'vim anyfile.txt' then quit, then kitty scrolls down normally if I do `ls -l` again

**Comment by page-down** [comment]
If you installed from the official repository, then kitty 0.15 is two years old. Please try installing the latest release to see if the issue still exists.

https://sw.kovidgoyal.net/kitty/binary/

**Comment by laoshaw** [comment]
the new version did fix the problem
